lucien-ios/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LoginManager.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/29/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+final class LoginViewModel {
+
+    let lucienAPIClient = LucienAPIClient()
```

Comment:
Even though this works, to make this class testable, create an initializer that takes a `LucienAPIClient` and actually do the API client creation in `RootViewController`. That allows us to pass in a mock version when we test and makes sure we don't have too many instances of the API client running around